tests/migration_test.go

```diff
@@ -230,7 +230,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
 			return err
 		}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
-		By("Waiting until the Migration Completes")
+		By("Waiting until the Migration Completes zzz")
```

Comment:
zzz?

Reply:
i will remove it of course, its still WIP, 
its a way for me to find fast the place where i want to add sleep